pallets/asset/src/lib.rs

```diff
@@ -300,6 +300,34 @@ pub mod pallet {
 
 			Ok(())
 		}
+/// Creates a new asset entry within a specified space.
```

Comment:
Please move the method documentation to top of the actual method which the doc is targeted for & match its indentation.

Reply:
Please review it again, as I have updated the documentation accordingly.